services/rds/src/main/java/org/finra/gatekeeper/services/accessrequest/model/AccessRequest.java

```diff
@@ -52,7 +53,8 @@
      */
 
     @Id
-    @GeneratedValue
+    @SequenceGenerator(name="hibernate_generator", sequenceName="hibernate_sequence")
+    @GeneratedValue(generator="hibernate_generator")
```

Comment:
i'm a little concerned with this change. I don't know how it will affect folks with existing request data. Seems risky.